nemo.yaml

```diff
@@ -0,0 +1,78 @@
+package:
+  name: nemo
+  version: 1.22.0
+  epoch: 0
+  description: conversational AI toolkit built for researchers working on ASR, TTS, LLMs, and NLP.
+  copyright:
+    - license: Apache-2.0
+  target-architecture:
+    - x86_64
+  options:
+    no-provides: true
+    no-depends: true
+  dependencies:
+    runtime:
+      - libgomp
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - ffmpeg-dev
+      - libsndfile-dev
+      - py3.11-installer
+      - py3.11-pip
+      - python-3.11
+      - python-3.11-dev
+      - samurai
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/NVIDIA/NeMo.git
+      tag: v${{package.version}}
+      expected-commit: 0b7467ebd158ef6876dd147189517b25ef626982
+
+  - runs: git submodule update --init --recursive
+
+  - runs: |
+      sed -i '/fasttext/d' requirements/requirements_nlp.txt
+      sed -i 's/megatron_core==0.4.0/megatron_core==0.3.0/' requirements/requirements_nlp.txt
+
+  # Build deps
+  - runs: |
+      # Setup the virtualenv
+      python -m venv .venv --system-site-packages
+      # Bump pip to patch a CVE
+      .venv/bin/pip install --upgrade wheel pip==23.3.2 setuptools==65.5.1
+      .venv/bin/pip install Cython
+      source .venv/bin/activate
+      ./reinstall.sh
+      pip install ".[all]"
+
+  - runs: |
+      echo ${{targets.destdir}}
+      mkdir -p ${{targets.destdir}}/usr/share/nemo
+      mv .venv ${{targets.destdir}}/usr/share/nemo
+
+  - runs: |
+      # edit the venv paths
+      rm -rf ${{targets.destdir}}/usr/share/nemo/.venv/bin/__pycache*
+      sed -i "s|/home/build|/usr/share/nemo|g" ${{targets.destdir}}/usr/share/nemo/.venv/bin/*
+
+  - runs: |
+      # allow site-packages
+      sed -i "s|include-system-site-packages = false|include-system-site-packages = true|g" ${{targets.destdir}}/usr/share/nemo/.venv/pyvenv.cfg
+
```

Comment:
Do we need an extra step here similar to this package?
- https://github.com/wolfi-dev/os/blob/9e21b9522815e253e6fe89102a999186d1e06c49/az.yaml#L51

i.e something like:

```
  - runs: |
      mkdir -p ${{targets.destdir}}/usr/bin
      ln -s /usr/share/nemo/.venv/bin/nemo ${{targets.destdir}}/usr/bin/nemo
```

However if i build nemo and check in the `/usr/share/nemo/.venv/bin` directory, i don't see a 'nemo' executable. if I do the same with the AZ package I do.